openQA.spec

```diff
@@ -159,8 +173,8 @@ The openQA worker manages test engine (provided by os-autoinst package).
 %package client
 Summary:        Client tools for remote openQA management
 Group:          Development/Tools/Other
-Requires:       openQA-common = %{version}
 Requires:       %client_requires
+Requires:       openQA-common = %{version}
```

Comment:
is this because of `spec-cleaner` or something?

Reply:
the reordering? yes, it sorts macros first